media-source-respec.html

```diff
@@ -142,7 +142,9 @@
     <section id="introduction" class="informative">
       <h2>Introduction</h2>
       <p>This specification allows JavaScript to dynamically construct media streams for &lt;audio&gt; and &lt;video&gt;. It defines a MediaSource object that can serve as a source of media data for an HTMLMediaElement. MediaSource objects have one or more <a>SourceBuffer</a> objects.  Applications append data segments to the <a>SourceBuffer</a> objects, and can adapt the quality of appended data based on system performance and other factors. Data from the <a>SourceBuffer</a> objects is managed as track buffers for audio, video and text data that is decoded and played. Byte stream specifications used with these extensions are available in the byte stream format registry [[MSE-REGISTRY]].</p>
-      <img src="pipeline_model.svg" alt="Media Source Pipeline Model Diagram" longdesc='pipeline_model_description.html#pipelinedesc'>
+      <a href='pipeline_model_description.html#pipelinedesc'>
+        <picture><img src="pipeline_model.svg" alt="Media Source Pipeline Model Diagram"></picture>
```

Comment:
`<a href>` replaces `longdesc` which is currently no-op on every browser. [`<picture>` workarounds ReSpec width=150 issue](https://github.com/w3c/respec/issues/3435).